typescript/web/src/connectors/repository/label.ts

```diff
@@ -6,7 +6,12 @@ import { list } from "./utils/list";
 /* `count` and `list` need to handle a specific logic when you want it to be filtered by dataset
  * We can't do joins with dexies so we need to do it manually. */
 
-export const countLabels = async (where?: LabelWhereInput) => {
+export const countLabels = async (
+  whereWithUser?: LabelWhereInput & { user?: { id: string } }
+) => {
+  const { user, ...wherePossiblyEmpty } = whereWithUser ?? {};
+  const where =
+    Object.keys(wherePossiblyEmpty).length < 1 ? null : wherePossiblyEmpty;
   if (where) {
     if ("datasetId" in where) {
```

Comment:
And same here, should we check if the user has access to the dataset?

Reply:
In this case we don't need it as we are in the local repository and we assume that the user has access to its local datasets